paparazzi/libs/build.gradle

```diff
@@ -0,0 +1 @@
+ext.repoDir = new File("$rootProject.buildDir/prebuilts/studio/layoutlib")
```

Comment:
move to top of buildscript block in `paparazzi/build.gradle`

Reply:
it's only used inside the lib folder

Reply:
correct, but libs isn't a gradle module

Reply:
It is, it just has no plugins applied, nor source code, only a build script.

Note: every `include(":a:b")` implicitly does an `include(":a")` otherwise, this wouldn't work.

See https://docs.gradle.org/current/dsl/org.gradle.api.initialization.Settings.html#org.gradle.api.initialization.Settings:include(java.lang.String[])
> As an example, the path a:b adds a project with path `:a:b`, name b and project directory $rootDir/a/b. It also adds the a project with path `:a`, name a and project directory $rootDir/a, if it does not exist already.